### DISCUSSION TITLE
Is write through fuse mode supported?

### DISCUSSION THREAD

**Discussion by edenbuaa** [juicedata/juicefs#5112] [General]
### Discussed in https://github.com/juicedata/juicefs/discussions/5063

<div type='discussions-op-text'>

<sup>Originally posted by **ddorian** August  6, 2024</sup>
Hi,

Is write through fuse mode supported (https://www.kernel.org/doc/Documentation/filesystems/fuse-io.txt) ? 
The docs only mention write-back.

If yes, how to enable it?</div>

**Comment by jiefenghuang** [comment]
write-through mode is the default.
